### ISSUE TITLE
Renewal fails with version 1.0.11

### ISSUE THREAD

**Issue by sfayer** [sfayer/cert_sorcerer#10]
```
Traceback (most recent call last):
  File "/usr/bin/CS.py", line 830, in <module>
    CS_UI.renew_cert(store, cn, hostcert)
  File "/usr/bin/CS.py", line 697, in renew_cert
    CS_CertTools.create_csr(store, cn, hostcert)
  File "/usr/bin/CS.py", line 289, in create_csr
    raise Exception("Certificate in wrong state to create new CSR.")
```

**Comment by sfayer** [comment]
This is a typo in only the python2 version introduced in 1.0.11: Will issue 1.0.12 with fix.

**Comment by sfayer** [comment]
Fixed in [1.0.12](https://github.com/sfayer/cert_sorcerer/releases/tag/v1_0_12), https://github.com/sfayer/cert_sorcerer/commit/3e979b04b99ae6f49e29b54f5fa19f0a14deaf51
